2996_reflection/reflection.md

```diff
@@ -3610,19 +3640,58 @@ Add another new paragraph defining _plainly constant-evaluated_ expressions:
     * during template argument deduction ([temp.deduct]{.sref}),
     * in a `$concept-id$` ([temp.names]{.sref}), or
     * in a `$requires-expression$` ([expr.prim.req]{.sref}), or
-  * [#.#.#]{.pnum} is a manifestly constant-evaluated initializer of a variable that is neither  `constexpr` ([dcl.constexpr]{.sref}) nor `constinit` ([dcl.constinit]{.sref}).
+  * [#.#.#]{.pnum} is, or is a subexpression of, an initializer for a variable that is neither  `constexpr` ([dcl.constexpr]{.sref}) nor `constinit` ([dcl.constinit]{.sref}).
```

Comment:
So like:
```cpp
void f() {
    std::string_view s = identifier_of(^^C); // not plainly constant-evaluated?
}
```
I think we should add an example here and mark expressions as being plainly constant-evaluated or not.

Reply:
Added a few, let me know what you think (or if you think any others are needed).